static/js/admin/vue-cdr-admin/tests/unit/chompb/preIngest.spec.js

```diff
@@ -0,0 +1,56 @@
+import { shallowMount } from '@vue/test-utils';
+import preIngest from '@/components/chompb/preIngest.vue';
+
+let wrapper;
+
+const project_info = [
+    {
+        "name": "file_source_test",
+        "createdDate": "2024-07-16T15:20:08.579384Z",
+        "creator": "bbpennel",
+        "indexedDate": "2024-08-15T12:38:28.662654Z",
+        "sourceFilesUpdatedDate": "2024-08-14T17:06:34.182417Z",
+        "groupMappingsUpdatedDate": "2024-08-14T17:42:49.626664Z",
+        "sipsSubmitted": [],
+        "bxcEnvironmentId": "test",
+        "projectSource": "files",
+        "status": "sources_mapped",
+        "allowedActions": ["color_bar_crop", "color_bar_report"],
+        "projectPath": "/opt/data/cdm_migrations/file_source_test"
```

Comment:
since it's a public repo i'd rather not have real server paths, could just make it `/path/to/file_source_test`. Same in the next example